docs/rollout/in-product-onboarding.md

```diff
@@ -0,0 +1,32 @@
+---
+title: In-product Onboarding
+layout: rollout
+---
+
+## Prefer in-product onboarding
+
+Tilt is a powerful tool, and thus requires some investment on the part of a new user to learn the basics. We recommend providing a [transition guide](../rollout/introduce-tilt) to help developers with onboarding. Ideally, many of the onboarding steps can be driven by _in-product onboarding_ instead, where Tilt itself guides the new user through initial set up, and usage of features, from basic to advanced.
```

Comment:
Love linking to the transition guide.

I think you'd still want much of what's in a transition guide even with in-product onboarding: describing why it's better, etc.

Reply:
Fixed. I made in-product onboarding complementary to the transition guide, not replacing it.